luigi/scheduler.py

```diff
@@ -885,6 +895,25 @@ def add_task(self, task_id=None, status=PENDING, runnable=True,
             self._state.get_worker(worker_id).tasks.add(task)
             task.runnable = runnable
 
+        return {
```

Comment:
So now this function returns something?

Reply:
Yeah, It returns almost the same variables as it takes (in slightly different format).
I needed only the `status` but I thought it makes better sense to include as much serialisable task as possible for completeness's sake.